GTBitsOfGood/national-npp
# Sahiti/62 profile photo uploads

## What is this?

- Profile photo is pre-populated on page load if the user has one
- Clicking on the profile avatar opens up file selection
- Selecting a new photo changes the avatar to the photo, but does not upload it
- Clicking "Save Changes" uploads the photo and stores the path to the photo on the user


## Setup


## Steps to Test

- Attach a chapter to your user
- Go to profile and ensure you are properly able to upload a profile image
- Do the same after attaching a nonprofit to your user

## Associated Issues

closes #62
